blob-stream/ionic-node
# Fileload

file download & load working.
Please try with 
- Android
- iOS
  To test do following:
- Go to stream
- click on download
- restart the app
- click on load
  you should see an image.
